spec/OBD/OBD.vspec

```diff
@@ -200,7 +200,7 @@ OxygenSensorsIn4Banks:
   type: sensor
   description: PID 1D - Presence of oxygen sensors in 4 banks. Similar to PID 13, but [A0..A7] == [B1S1, B1S2, B2S1, B2S2, B3S1, B3S2, B4S1, B4S2]
 
-AuxInputStatus:
+IsPTOActive:
```

Comment:
PTO?

Reply:
Looking at https://en.wikipedia.org/wiki/OBD-II_PIDs the description of  `#1E` is Auxiliary input status, but  looking at the data only 1 bit is used and it is used for Power Take Off (PTO).  (see https://en.wikipedia.org/wiki/Power_take-off)  

```
A0 == Power Take Off (PTO) status (1 == active)
[A1..A7] not used
```

To make a good boolean name I think the name shall be based on PTO in some way, either like this or expanded like IsPowerTakeOffActtive. The current description gives some hints that PTO ="power take off" but that could of course be improved.

`PID 1E - Auxiliary input status (power take off)`